hypnotoad/core/mesh.py

```diff
@@ -2531,11 +2632,12 @@ def makeRegions(self):
                 ), "region should not have been added to any yGroup before"
                 region.yGroupIndex = len(group)
                 group.append(region)
-                region_set.remove(region)
+                region_list.pop(i)
```

Comment:
I guess this is not a significant change from the previous code, but we should be very careful modifying lists while iterating over them. It might be better to make a copy and `pop` elements from that?